### PR TITLE
Feature/unprivileged user

### PR DESCRIPTION
**Pull request by hostmaster**
A separate branch is required to support two flavors.
- running with root privileges (can bind to 443, no need for external SSL termination) `:latest` tag
- unprivileged user `:unprivileged-user` tag

@jtsoi please keep  the branch open
@vs-ab 

I setup an automated build from master and the branch.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+NAME = virtusize/docker_oauth2_proxy
+VERSION ?= unprivileged-user
+
+.PHONY: all build
```
Comment (ghost) [inline Makefile]: - push?

**Comment by ghost** [comment]
:+1:

**Comment by jtsoi** [comment]
OK

**Comment by ghost** [comment]
@hostmaster what's the status of this PR?

**Comment by hostmaster** [comment]
@vs-ab It is not a PR. I'm using this branch for wiki

**Comment by ghost** [comment]
@hostmaster ah, I see. Is this something that can't or shouldn't be merged?

**Comment by hostmaster** [comment]
exactly

**Comment by ghost** [comment]
Got it. In that case I suggest to close this PR, but keep the branch.

**Pull request closed without merging**